src/handlers/schedule-checkin.ts

```diff
@@ -203,9 +210,13 @@ function isRequestBody(value: any): value is RequestBody {
   );
 }
 
+interface CheckinTime {
```

Comment:
Any better names for this? I struggled figuring out a good name.

Also, is this what you meant by an array of objects?

Reply:
This is exactly what I meant, looks great! Hard to name something like this, but this name makes sense to me.